src/main/java/sg/ncl/MainController.java

```diff
@@ -741,8 +741,21 @@ public String userSideAcceptJoinRequest(@PathVariable String teamId, @PathVariab
     }
     
     @RequestMapping("/approve_new_user/reject/{teamId}/{userId}")
-    public String userSideRejectJoinRequest(@PathVariable Integer teamId, @PathVariable Integer userId) {
-        teamManager.rejectJoinRequest(userId, teamId);
+    public String userSideRejectJoinRequest(@PathVariable String teamId, @PathVariable String userId, HttpSession session) {
+        JSONObject mainObject = new JSONObject();
+        JSONObject userFields = new JSONObject();
+
+        userFields.put("id", session.getAttribute("id").toString());
+        mainObject.put("user", userFields);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        headers.set("Authorization", AUTHORIZATION_HEADER);
+
+        HttpEntity<String> request = new HttpEntity<String>(mainObject.toString(), headers);
+        ResponseEntity responseEntity = restTemplate.exchange(properties.getRejectJoinRequest(teamId, userId), HttpMethod.DELETE, request, String.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "responseEntity" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "responseEntity". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)